### ISSUE TITLE
Index has disappeared from PDF for Japanese documents at v1.8.0

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#5454]
### Problem

Japanese projects have lost index in PDF output since Sphinx v1.8.0.

#### Procedure to reproduce the problem

Build PDF for this

```
Welcome to FOO's documentation!
===============================


Section
=======

:index:`aaa`

:index:`目次`
```

with `language` set to `'ja'`

The PDF contains no index.

When built with Sphinx v1.7.9 there is index in PDF.

First "bad" commit appears to be ba97fd4f8d0f6c82ba44f2298093a040bde28c51

### Environment info
- OS: Mac
- Python version: 3.6.5
- Sphinx version: v1.8.0
- TeXLive 2018

**Comment by jfbu** [comment]
The commit https://github.com/sphinx-doc/sphinx/commit/ba97fd4f8d0f6c82ba44f2298093a040bde28c51 tries to improve Windows compatibility by using Perl syntax for file operations rather than Unix syntax. But for many Windows users, Perl is not installed anyhow, so I think we could keep Unix syntax and move all Windows customization to texinputs_win rather. Sadly, I can not help at all with Windows...

**Comment by tk0miya** [comment]
Fixed by #5465 
Thank you for reporting :-)
